src/utils/fetchCreateTokenEvents.ts

```diff
@@ -36,53 +36,58 @@ async function findBlock(blockNumber: number): Promise<BlockI> {
 }
 
 export async function getTokenCreatedEvents() {
-  logger.info('==== Start event import ====')
-  const contract = new ethers.Contract(contractAddress, abi, provider)
+  try {
+    logger.info('==== Start event import ====')
+    const contract = new ethers.Contract(contractAddress, abi, provider)
 
-  const filter = 'TokenCreated'
-  const startBlock = await getLatestEventBlockNumberFromDb()
-  const endBlock = 'latest'
-  const events = await contract.queryFilter(filter, startBlock, endBlock)
-  const cleanedEvents: CreateTokenEventI[] = []
+    const filter = 'TokenCreated'
+    const startBlock = (await getLatestEventBlockNumberFromDb()) ?? 0
+    const endBlock = 'latest'
+
+    const events = await contract.queryFilter(filter, startBlock, endBlock)
+    const cleanedEvents: CreateTokenEventI[] = []
+
+    for (const event of events) {
+      const { blockNumber, transactionHash }: { blockNumber: number; transactionHash: string } = event
+      const existingEvent = await findCreateTokenEvent(transactionHash)
+      const eventBlock = await findBlock(blockNumber)
+      if (!eventBlock && !existingEvent) {
+        cleanedEvents.push({
+          blockNumber,
+          unixTimestamp: 1234567890,
+          timestamp: getDateFromUnixTimestamp(1234567890),
```

Comment:
question(blocking): is there a way to unify `timestamp` and `unixTimestamp`?

Reply:
My thought process here is to have easy access to both formats without having to convert all the time if you work with the large amount of data. The tradeoff is one conversion during fetching and some minor space costs. 

The other idea was to keep only the converted `Date` format, but I wanted to have the original value saved as well.